projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -99,11 +98,12 @@ public function reregister_menu_items() {
 		$this->add_testimonials_menu( $wp_admin );
 		$this->add_portfolio_menu( $wp_admin );
 		$this->add_comments_menu( $wp_admin );
-		$this->add_appearance_menu( $wp_admin );
-		$this->add_plugins_menu();
+		$this->add_appearance_menu( $wp_admin, $wp_admin );
```

Comment:
using `$wp_admin` two times feels a little awkward. I can understand why this happens, but is there chance we can make things more declarative?

Reply:
71d945b